js/thumbnail.js

```diff
@@ -159,7 +159,7 @@ function Thumbnail (fileId, square) {
 		/**
 		 * Returns the link to the media type icon
 		 *
-		 * Modern browsers get an SVG, older ones a PNG
+		 * All browsers get an SVG always
```

Comment:
Just remove that  line and change the 1st line to
"Returns the link to the SVG media type icon"